<launch>
  <rosparam command="load" ns="roll_controller" file="$(find riptide_controllers)/cfg/roll_config.yaml" />
  <rosparam command="load" ns="pitch_controller" file="$(find riptide_controllers)/cfg/pitch_config.yaml" />
  <rosparam command="load" ns="yaw_controller" file="$(find riptide_controllers)/cfg/yaw_config.yaml" />
  <node pkg="riptide_controllers" type="attitude_controller" name="attitude_controller" output="screen" />
</launch>
